### PR TITLE
Feature/submission form

### PR DESCRIPTION
**Pull request by mr-ryan12**
## Checklist
- [ ] Tests written where required
- [ ] All tests passing
- [ ] Unused code/prys/etc deleted
Testing Coverage at <percentage>%

## Changes Made
  
What (if any) features are you implementing?
- Created a submission form

What (if anything) did you refactor?
- Nothing at the time

Were there any issues that arose?
- Researching uncontrolled form inputs

Is there anything that you need from your teammate?
- PR reviews and merge

Any other comments, questions, or concerns?
- Not at the moment

### CODE REVIEW

**Review by Echory — APPROVED** [review]
This looks really good Ryan thanks for doing this! Interesting how we handle the value of the input.

**Review by oacortez — APPROVED** [review]
Thank you for creating this branch, Ryan! The submission for the form looks perfect, I like how the users are able to see their profile image before they submit. Something to consider looking is the file size of the video, other than that everything looks good.

File: src/components/SubmissionForm/SubmissionForm.js [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+  const handleSongTitle = event => {
+    setSongTitle(event.target.value)
+  }
+  const handleVideo = event => {
+    setVideo(event.target.value)
+  }
+  const handleProfileImage = event => {
+    setProfileImage(URL.createObjectURL(event.target.files[0]))
+  }
+  const profilePicturePreview = profileImage ? <img src={profileImage} alt="Profile picture logo" className="profile-picture-preview"/>
```
Comment (aliroemhildt) [inline src/components/SubmissionForm/SubmissionForm.js:33]: This is super cool, great work Ryan!

**Review by aliroemhildt — APPROVED** [review]
Some seriously great work, thanks Ryan!

**Pull request merged**